sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/UrlHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    internal class UrlHelper
+    {
+        private const string SchemePostfix = "://";
+        private const string Colon = ":";
+        private const string HttpPort80 = "80";
```

Comment:
Same here, I think the benefit of having a const here is not paying off?

Reply:
`GetUrl` is in the hot path and gets executed for all HTTP activity. This is reused for most the times.